vault/identity_store_groups_test.go

```diff
@@ -118,11 +118,16 @@ func TestIdentityStore_MemDBGroupIndexes(t *testing.T) {
 	}
 
 	// Insert it into memdb
-	err = i.MemDBUpsertGroup(group)
+	txn := i.db.Txn(true)
+
+	err = i.MemDBUpsertGroupInTxn(txn, group)
 	if err != nil {
+		txn.Abort()
```

Comment:
Optional: This works as well, but we could just defer like it's doing elsewhere with txns.

Reply:
We are changing the reference to `txn` below in the same function. So I worry if the defer would not do the right thing.

Reply:
I see what you mean. I did a [quick test](https://play.golang.org/p/4r4hyUwjZi_X) to see what's the behavior and it _seems_ fine, but it might actually be cleaner to keep it as-is.